src/forms/__tests__/mappers-test.js

```diff
@@ -33,4 +33,13 @@ describe('mappers', () => {
       expect(specialPhoneMapper.fromRaw('22-45')).toEqual({ ext: 22, num: 45 });
     });
   });
+
+  describe('booleanMapper', () => {
+    it('toRaw should map true to true', () => {
+      expect(booleanMapper.toRaw(true)).toEqual(true);
+    });
+    it('fromRaw should map true to true', () => {
+      expect(booleanMapper.fromRaw(true)).toEqual(true);
```

Comment:
What about falsy values and values like string and numbers? Keep in mind that a test is also a reference implementation that can be used to understand the codes behaviour in the future.